src/app/modules/main/profile_section/wallet/networks/view.nim

```diff
@@ -0,0 +1,66 @@
+import Tables, NimQml, sequtils, sugar
+
+import ../../../../../../app_service/service/network/dto
+import ./io_interface
+import ./model
+import ./item
+
+QtObject:
+  type
+    View* = ref object of QObject
+      delegate: io_interface.AccessInterface
+      networks: Model
+      areTestNetworksEnabled: bool
+
+  proc setup(self: View) =
+    self.QObject.setup
+
+  proc delete*(self: View) =
+    self.QObject.delete
+
+  proc newView*(delegate: io_interface.AccessInterface): View =
+    new(result, delete)
+    result.delegate = delegate
+    result.networks = newModel()
+    result.setup()
+
+  proc areTestNetworksEnabledChanged*(self: View) {.signal.}
+
+  proc getAreTestNetworksEnabled(self: View): QVariant {.slot.} =
```

Comment:
@alaibe this could be just `bool`, no need to wrap it to `QVariant`.

Reply:
I had undefined error in qml when doing this

Reply:
If so, then ok.

Reply:
ohh also changing the [bool] will try

Reply:
it worked!